scene/animation/animation_player.cpp

```diff
@@ -1203,11 +1203,15 @@ void AnimationPlayer::play(const StringName &p_name, float p_custom_blend, float
 	}
 
 	_stop_playing_caches();
-
+	
```

Comment:
There's now a trailing tab here that should be removed.

Reply:
Removed.